src/main/webapp/components/results/result-index-cards.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { IndexCardItem, Actions } from '../index-cards'
+import AddIcon from '@material-ui/icons/Add'
+import GetAppIcon from '@material-ui/icons/GetApp'
+import IconButton from '@material-ui/core/IconButton'
+
+export default props => {
+  const { results, handleOpen, setResult } = props
+  return React.useMemo(
+    () =>
+      results.map(({ metacard }) => (
+        <IndexCardItem
+          key={metacard.attributes.id}
+          title={metacard.attributes.title}
+          subHeader={' '}
+          titleTypographyProps={{
+            variant: 'body1',
```

Comment:
![image](https://user-images.githubusercontent.com/36056781/78876579-67506000-7a04-11ea-922d-4b9ade5950c4.png)
Thoughts on this styling change

Reply:
lgtm - on an unrelated note, it kind of looks like the add and download buttons are slightly off center, but that could just be the way the SVGs are